ci/travis/ubuntu_1804/script.sh

```diff
@@ -32,6 +32,9 @@ rm autotest/ogr/ogr_fgdb.py
 
 (cd autotest/ogr && $PYTEST ogr_mssqlspatial.py)
 
+# HANA tests
+(cd autotest/ogr && OGR_HANA_CONNECTION_STRING="DRIVER={\'/usr/sap/hdbclient/libodbcHDB.so\'};HOST=917df316-4e01-4a10-be54-eac1b6ab15fb.hana.prod-us10.hanacloud.ondemand.com;PORT=443;USER=QGISCI;PASSWORD=tQ&7W3Klr9!p" $PYTEST ogr_hana.py)
```

Comment:
@rouault, I was not able to find where the OGR_HANA_CONNECTION_STRING variable needs to be set on Ubuntu_20.04. I tried to follow OGR_MYSQL_CONNECTION_STRING/OGR_PG_CONNECTION_STRING values, but couldn't find anything. For Ubuntu_18.04, it is absolutely obvious. Could you point me to the right location?

Reply:
as an extra -e name=value argument to the docker run invokation of the "Run tests" step of the .yml file ?

Reply:
Thank you, Even. Done in 7c3bfcaa057f85c3abfaadff43a28879c2222b0a.